data/packages/com.mlapi.contrib.transport.litenetlib.yml

```diff
@@ -1,16 +1,16 @@
 name: com.mlapi.contrib.transport.litenetlib
 displayName: LiteNetLib Transport for MLAPI
 description: LiteNetLib Transport for MLAPI
-repoUrl: 'https://github.com/Unity-Technologies/multiplayer-community-contributions'
+repoUrl: 'https://github.com/Unity-Technologies/multiplayer-community-contributions/tree/main/Transports/com.mlapi.contrib.transport.litenetlib'
```

Comment:
The repoUrl shouldn't include the path. And OpenUPM pipelines will detect the package path automatically, no need to specify the path.